src/bin/mosaic-stop.ts

```diff
@@ -6,15 +6,17 @@ import NodeDescription from '../Node/NodeDescription';
 import Node from '../Node/Node';
 import GraphDescription from '../Graph/GraphDescription';
 import Graph from '../Graph/Graph';
+import Utils from '../Utils';
 
 mosaic
   .arguments('<chains...>')
   .action((chains: string[]) => {
     // Chain can't be validated as origin chain id is not received for aux chain.
     for (const chain of chains) {
-      const node: Node = NodeFactory.create(new NodeDescription(chain));
+      const chainId = Utils.getChainId(chain);
+      const node: Node = NodeFactory.create(new NodeDescription(chainId));
       node.stop();
-      const graph: Graph = new Graph(new GraphDescription(chain));
+      const graph: Graph = new Graph(new GraphDescription(chainId));
```

Comment:
`mosaic <command> 1000` is also working along with `mosaic <command> dev-auxiliary`.
is this expected?

Reply:
I think, it's fine. Both represents aux chain.